ui-particles-angular/projects/ui-particles-angular/src/scss/mat-override/mat-input.stories.ts

```diff
@@ -0,0 +1,184 @@
+ * limitations under the License.
+ */
+import { MatInputModule } from '@angular/material/input';
+import { GioIconsModule } from 'ui-particles-angular';
+import { MatIconModule } from '@angular/material/icon';
+import { Story } from '@storybook/angular/types-7-0';
+import { Meta, moduleMetadata } from '@storybook/angular';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MatChipInputEvent, MatChipsModule } from '@angular/material/chips';
+import { FormControl, ReactiveFormsModule, Validators } from '@angular/forms';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatNativeDateModule } from '@angular/material/core';
+
+export default {
+  title: 'Material Override',
+  decorators: [
+    moduleMetadata({
+      imports: [
+        MatInputModule,
+        MatIconModule,
+        GioIconsModule,
+        MatChipsModule,
+        MatDatepickerModule,
+        MatNativeDateModule,
+        BrowserAnimationsModule,
+        ReactiveFormsModule,
+      ],
+    }),
+  ],
+  render: () => ({}),
+} as Meta;
+
+export const MatInput: Story = {
+  render: () => ({
+    template: `
+        <h3>Design System inputs</h3>
+        
+        <div class="container">
+          <mat-form-field appearance="outline">
+            <mat-label>Simple input</mat-label>
+            <input matInput>
+          </mat-form-field>
+          <mat-form-field appearance="outline">
+            <mat-icon svgIcon="gio:star-outline" matPrefix></mat-icon>
+            <mat-label>Simple input with icon</mat-label>
+            <input matInput>
+          </mat-form-field>
+          <mat-form-field appearance="outline">
+            <mat-icon svgIcon="gio:star-outline" matPrefix></mat-icon>
+            <mat-label>Simple input with icon and hint</mat-label>
+            <input matInput>
+            <mat-hint>This is a hint text</mat-hint>
+          </mat-form-field>
+          <mat-form-field appearance="outline" >
+            <mat-icon svgIcon="gio:star-outline" matPrefix></mat-icon>
+            <mat-label>Click to see control with error</mat-label>
+            <input matInput required [formControl]="errorFormControl">
+            <mat-error *ngIf="errorFormControl.hasError('required')">This is an error message</mat-error>
+          </mat-form-field>
+          
+          <mat-form-field appearance="outline">
+            <mat-label>Number input</mat-label>
+            <input matInput type="number">
+          </mat-form-field>
+          
+          <mat-form-field appearance="outline">
+            <mat-label>Color input</mat-label>
+            <input matInput type="color">
+          </mat-form-field>
+          
+          <mat-form-field appearance="outline">
+            <mat-label>Password input</mat-label>
+            <input matInput type="password">
+          </mat-form-field>
+
+          <mat-form-field class="example-chip-list" appearance="outline">
```

Comment:
Maybe we can use gio-form-tags-input ? normally we never use this component directly because it allows some "weird" stuff like selection with space 🤷‍♂️